app/javascript/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
```

Comment:
- [ ] Kindly install the `redux` and `redux-thunk` packages. Your app does not run without them.